packages/vscode-extension/README.md

```diff
@@ -51,15 +50,18 @@ Verify you have the right prerequisites for building Teams apps and install some
 
 ### Create your project
 
-Use the Teams Toolkit in Visual Studio Code to set up your first app project. Create your app project using the following steps:
+Use the Teams Toolkit for Visual Studio Code to set up your first app project. Create your tab app project using the following steps:
 
-- Ensure you've installed the Microsoft Teams Toolkit
-- In Visual Studio Code, in `DEVELOPMENT` section click `Create a new Teams app`. Or open the Command Palette (Ctrl+Shift+P / ⌘⇧-P or View -> Command - Palette) and type "teams" and choose `Teams: Create New Project`.
-- Next, Add capabilities comes up. Select capability you want then Next.
-- Choose options according to your purpose, i.e., front-end hosting type, language, cloud resources and so on.
+- Ensure you've installed the Microsoft Teams Toolkit for Visual Studio Code
+- Select the Teams Toolkit icon in the Visual Studio Code sidebar.
+- Select `Create a new Teams app` button and at next prompt to also select `Create a new Teams app`.
+- Ensure that Tab is selected as the capability that you want to build in your app. Select OK.
+- Select JavaScript as the programming language.
```

Comment:
should be "select the language you prefer"

Reply:
as explained. making the first section the simplest possible experience, as we've inserted the Read More link to the official user manual doc.